examples/actix-http/Cargo.toml

```diff
@@ -6,7 +6,8 @@ publish = false
 
 [dependencies]
 opentelemetry = { path = "../../opentelemetry", features = ["rt-tokio"] }
-opentelemetry-jaeger = { path = "../../opentelemetry-jaeger", features = ["reqwest_collector_client", "rt-tokio-current-thread"] }
+opentelemetry_sdk = { path = "../../opentelemetry-sdk"}
+opentelemetry-otlp = { path = "../../opentelemetry-otlp", features = ["tonic"] }
```

Comment:
Not sure if this is right but I don't seem to see `tonic` as a listed [feature](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-otlp/Cargo.toml#L61)?